src/createServer.js

```diff
@@ -1,11 +1,50 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const {
+  getUsers,
+  getUser,
+  deleteUser,
+  updateUser,
+  postUser,
+} = require('./controllers/users-controller.js');
+const {
+  getExpenses,
+  postExpense,
+  getExpense,
+  deleteExpense,
+  patchExpense,
+} = require('./controllers/expenses-controller.js');
+const { resetStoredData } = require('./helpers/resetStoredData.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
```

Comment:
You can apply middlewares globally like that. Thanks to that you dont have to add middleware on every endpoint.

```suggestion
const app = express();
app.use(express.json());
```